sdk/servicebus/azure-servicebus/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * Fixed bug where sync AutoLockRenew does not shutdown itself timely.
 * Fixed bug where async AutoLockRenew does not support context manager.
+* Renamed `receive()`, `peek()` `schedule()` and `send()` to `receive_messages()`, `peek_messages()`, `schedule_messages()` and `send_messages()` to align with other service bus SDKs.
```

Comment:
This should be under "breaking changes"

Reply:
The things I will do to avoid stashing changes and switching branches. :P

```suggestion

**Breaking Changes**

* Renamed `receive()`, `peek()` `schedule()` and `send()` to `receive_messages()`, `peek_messages()`, `schedule_messages()` and `send_messages()` to align with other service bus SDKs.
```